apiserver/facades/client/client/status_test.go

```diff
@@ -795,23 +795,6 @@ func (s *statusUnitTestSuite) TestMachineWithNoDisplayNameHasItsEmptyDisplayName
 	c.Assert(status.Machines[machine.Id()].DisplayName, gc.Equals, "")
 }
 
-func (s *statusUnitTestSuite) TestMachineWithDisplayNameHasItsDisplayNameSent(c *gc.C) {
```

Comment:
I don't get why we're at liberty to remove the display name?

Reply:
As discussed offline, this was not part of the DDL and I have now added it. This test has now been restored.